awx/main/management/commands/run_heartbeet.py

```diff
@@ -50,17 +52,22 @@ def construct_payload(self, action='online'):
         }
         return json.dumps(payload)
 
+    def notify_listener_and_exit(self, *args):
+        with pg_bus_conn(new_connection=False) as conn:
+            conn.notify('web_heartbeet', self.construct_payload(action='offline'))
+        sys.exit(1)
```

Comment:
This is the normal way to exit the program, we probably should exit with 0 instead.